### PR TITLE
Test Plan for Day2Ops

### PR DESCRIPTION
**Pull request by malnick**
Logging:
- https://github.com/dcos/dcos/pull/1045

Debugging:
- https://github.com/dcos/dcos/pull/1031

Metrics
- https://github.com/dcos/dcos/pull/778

### CODE REVIEW

**Comment by adam-mesos** [comment]
For debugging, please pull in https://github.com/dcos/dcos/pull/1031

File: packages/dcos-integration-test/extra/test_mesos.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            'Got %s with POST request to %s with headers %s and data %s.',
+            r.status_code,
+            url,
+            headers,
+            data
+        )
+        assert r.status_code == 200
+        return r
+
+    def find_container_id(state, app_id):
```
Comment (darkonie) [inline packages/dcos-integration-test/extra/test_mesos.py]: @adam-mesos i think it's a way difficult to ready so many nested `if`
i have a function that does it already https://github.com/dcos/dcos/pull/1040/files#diff-0983fe52e111e591e8e619b14979ce14R105 but it returns a URL. We could probably refactor it and use the one implementation across tests.

Comment (adam-mesos) [inline packages/dcos-integration-test/extra/test_mesos.py]: Agreed. +1 to code reuse. I actually copied most of this from the cli code anyway. :)

**Comment by adam-mesos** [comment]
Note that we should also include https://github.com/dcos/dcos/pull/974 for the cosmos changes enabling the logging CLI.

**Comment by adam-mesos** [comment]
No longer needed. Please close @lingmann @malnick

**Pull request closed without merging**